client/state/memberships/subscriptions/actions.js

```diff
@@ -13,6 +17,64 @@ export const requestSubscriptionsList = () => ( {
 	type: MEMBERSHIPS_SUBSCRIPTIONS_LIST_REQUEST,
 } );
 
+export const requestAutoRenewDisable = ( subscriptionId ) => {
+	return ( dispatch ) => {
+		dispatch( {
+			type: MEMBERSHIPS_SUBSCRIPTION_UPDATING,
+			subscriptionId,
+		} );
+		return wpcom.req
+			.post( `/me/memberships/subscriptions/${ subscriptionId }/auto_renew/disable` )
+			.then( ( response ) => {
+				dispatch( {
+					type: MEMBERSHIPS_SUBSCRIPTION_UPDATE,
+					subscriptionId,
+					updates: response.subscription,
+				} );
+				dispatch( {
+					type: MEMBERSHIPS_SUBSCRIPTION_UPDATING_SUCCESS,
+					subscriptionId,
+				} );
+			} )
+			.catch( ( error ) => {
+				dispatch( {
+					type: MEMBERSHIPS_SUBSCRIPTION_UPDATING_FAILURE,
+					subscriptionId,
+					error,
+				} );
+			} );
+	};
+};
+
+export const requestAutoRenewResume = ( subscriptionId ) => {
```

Comment:
Neatpick: maybe a simple SwitchAutoRenew endpoint would have been enough, instead of this duplication (even on the back-end)